core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/common/Entity.java

```diff
@@ -36,15 +38,43 @@ public Entity() {
 
     }
 
-    public Entity(String displayName, String tenantDomain,
```

Comment:
Public API change?

Reply:
This was introduced by my new API so nobody has used. hence no issue